src/sentry/models/groupassignee.py

```diff
@@ -66,6 +67,39 @@ def get_assignee_data(self, assigned_to: Team | RpcUser) -> tuple[str, str, str]
 
         return (assignee_type, assignee_type_attr, other_type)
 
+    def remove_old_assignees(
+        self, group: Group, previous_assignee: Optional(GroupAssignee)
+    ) -> None:
+        if features.has("organizations:participants-purge", group.organization):
+            if (
+                features.has("organizations:team-workflow-notifications", group.organization)
+                and previous_assignee.team_id
+            ):
+                GroupSubscription.objects.filter(
```

Comment:
Wondering if it makes sense to keep a stat around the deletions here. Not sure how important, but might be helpful for monitoring?

Reply:
I added some logging